dt/host/InspectorFrontendHost.js

```diff
@@ -156,7 +157,8 @@ WebInspector.InspectorFrontendHostStub.prototype = {
      */
     copyText: function(text)
     {
-        WebInspector.console.error("Clipboard is not enabled in hosted mode. Please inspect using chrome://inspect");
+        const clipboard = require('clipboard');
```

Comment:
wonder if we should pull this to the top of the file or something vs inline require in the method